jobs/furnishings/src/furnishings/stage_processors/stage_one.py

```diff
@@ -95,6 +114,58 @@ def generate_paper_letters(self):
             self._app.logger.error(f'Error generating batch letters: {e}')
         self._app.logger.debug('Finish generating batch letters.')
 
+    def upload_to_sftp(self, client, data, filename):
```

Comment:
This is a generic function to SFTP data here. Trying to keep things as clean as possible here.